README.md

```diff
@@ -10,6 +10,20 @@ distributed systems and blockchain protocols. It combines the robust theoretical
 basis of the [Temporal Logic of Actions][TLA] (TLA) with state-of-the-art static
 analysis and development tooling.
 
+This is how a typical Quint code looks like in the [Quint VSCode Plugin][]:
+
+<p align="center">
+  <img src="./doc/img/validate-quint.png"
+       width="70%" title="Code snippet in Quint">
+</p>
```

Comment:
This ends up being very tiny on mobile:

![image](https://user-images.githubusercontent.com/19820422/218844990-adc2a542-d3f8-4fd7-9201-3aa787fe3b3c.png)

I wonder if it wouldn't be better to just use a codeblock here? I know we don't
have syntax highlighting on GitHub, but we could rig up a landing page using JS
highlighting once our priorities shift to publicity. This would give us the
benefit of accessibility and responsive scaling, at the cost of some superficial
color for the time being. WDYT?

Reply:
That's what I also thought. But I am not sure how to embed HTML code with JS into markdown. Github restricts HTML in Markdown.

Reply:
yeah, afaik, we can't to highlight.js or whatever on github's markdown rendering, but we could of course on a static site hosted from our gh page.